Building on Snow error re: SpinerEOSDependsRhoT not declared

I tried it with manual modules (gcc 7.4.0, openmpi 2, and cmake) and then with the machine configuration found in parthenon, but get the same error. I didn't notice any clone errors.

```
/users/woodring/projects/phoebus/phoebus/src/microphysics/eos_phoebus/eos_phoebu
s.cpp: In function ‘std::shared_ptr<parthenon::StateDescriptor> Microphysics::EO
S::Initialize(parthenon::ParameterInput*)’:
/users/woodring/projects/phoebus/phoebus/src/microphysics/eos_phoebus/eos_phoebu
s.cpp:54:28: error: ‘SpinerEOSDependsRhoT’ has not been declared
       IdealGas::EosType(), SpinerEOSDependsRhoT::EosType(),
                            ^~~~~~~~~~~~~~~~~~~~
/users/woodring/projects/phoebus/phoebus/src/microphysics/eos_phoebus/eos_phoebu
s.cpp:55:7: error: ‘SpinerEOSDependsRhoSie’ has not been declared
       SpinerEOSDependsRhoSie::EosType()};
       ^~~~~~~~~~~~~~~~~~~~~~
/users/woodring/projects/phoebus/phoebus/src/microphysics/eos_phoebus/eos_phoebu
s.cpp:55:40: error: could not convert ‘{singularity::IdealGas::EosType()(), <exp
ression error>, <expression error>}’ from ‘<brace-enclosed initializer list>’ to
 ‘const std::vector<std::__cxx11::basic_string<char> >’
       SpinerEOSDependsRhoSie::EosType()};
                                        ^
make[2]: *** [src/CMakeFiles/phoebus.dir/microphysics/eos_phoebus/eos_phoebus.cp
p.o] Error 1
make[1]: *** [src/CMakeFiles/phoebus.dir/all] Error 2
make: *** [all] Error 2
```

--- a/src/microphysics/eos_phoebus/eos_phoebus.cpp
+++ b/src/microphysics/eos_phoebus/eos_phoebus.cpp
@@ -51,8 +51,12 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   // EOSBuilder::params_t shifted_params, scaled_params;
 
   const std::vector<std::string> valid_eos_names = {
-      IdealGas::EosType(), SpinerEOSDependsRhoT::EosType(),
-      SpinerEOSDependsRhoSie::EosType()};
+      IdealGas::EosType()
+#ifdef SPINER_USE_HDF
+      , SpinerEOSDependsRhoT::EosType(),
+      SpinerEOSDependsRhoSie::EosType()
+#endif
+  };
   std::string eos_type = pin->GetString(block_name, std::string("type"));
   if (eos_type.compare(IdealGas::EosType()) == 0) {
     type = EOSBuilder::EOSType::IdealGas;

